src/Nethermind/Nethermind.Evm/ByteCodeBuilderExtensions.cs

```diff
@@ -0,0 +1,335 @@
+//  Copyright (c) 2021 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Nethermind.Core;
+using Nethermind.Core.Collections;
+using Nethermind.Core.Extensions;
+using Nethermind.Int256;
+using Nethermind.Serialization.Json;
+using Org.BouncyCastle.Asn1.Mozilla;
+
+namespace Nethermind.Evm
+{
+    /// <summary>
+    /// A utility class that extends Prepare class to abstract stack-like behaviour
+    /// </summary>
+    ///
+
+    //Note(Ayman) : redesign arg order to add flexibility and ability to use args already in stack, and add tests
```

Comment:
I believe that is already done. so delete the comment

Reply:
Yes it was resolved.